### PR TITLE
Fix syscall name and arguments printing on x86 windows guests

### PR DESCRIPTION
**Pull request by disaykin**

### CODE REVIEW

**Comment by tklengyel** [comment]
@drakvuf-jenkins This is OK to test

File: src/plugins/syscalls/win.cpp [hunk 1]
```diff
@@ -353,17 +355,17 @@ static bool trap_syscall_table_entries(drakvuf_t drakvuf, vmi_instance_t vmi, sy
...
 
-    if ( s->pm == VMI_PM_IA32E )
+    if ( !s->is32bit )
     {
         if ( !drakvuf_get_kernel_symbol_rva(drakvuf, "KiSystemServiceStart", &start) )
             throw -1;
 
         system_service_table_x64 sst[2] = {};
-        if ( VMI_FAILURE == vmi_read_ksym(lg.vmi, "KeServiceDescriptorTableShadow", 2*sizeof(system_service_table_x64), (void*)&sst, NULL) )
+        if ( VMI_FAILURE == vmi_read_ksym(vmi, "KeServiceDescriptorTableShadow", 2*sizeof(system_service_table_x64), (void*)&sst, NULL) )
```
Comment (tklengyel) [inline src/plugins/syscalls/win.cpp:368]: I'm surprised this works with `auto vmi`

**Comment by tklengyel** [comment]
Unfortunately this also fails the x86 CI test:
```
Running DRAKVUF #2 for 60 seconds. Opts:  -S /shared/jenkins/DRAKVUF-windows7-sp1-x86/ci/syscalls.txt -a syscalls
DRAKVUF is running with PID 233587

CPU utilization average: 0

Syscalls: 0

**Comment by disaykin** [comment]
@tklengyel This is very strange.

I touch the logic of finding of arguments definition. This is does not change trap adding and handling.

Maybe this is segfault. Can you please show me debug logs of run?

File: src/plugins/syscalls/win.cpp (hunk 1 above)
Comment (disaykin) [reply src/plugins/syscalls/win.cpp:368]: This is implicit cast operator magic of C++.

**Comment by tklengyel** [comment]
@drakvuf-jenkins Retest this please

**Comment by tklengyel** [comment]
@drakvuf-jenkins Retest this please

**Comment by tklengyel** [comment]
Seems to work now

File: src/plugins/syscalls/win.cpp (hunk 1 above)
Comment (tklengyel) [reply src/plugins/syscalls/win.cpp:369]: Very handy

**Pull request merged**